src/classes/ExpectSObject_Tests.cls

```diff
@@ -0,0 +1,117 @@
+@isTest
+public class ExpectSObject_Tests {
+    static String tooLong = 'Account*Name*Cannot*Be*More*Than*255*Characters=================================================================================================================================================================================================================';
```

Comment:
I think something like 'Limit255Chars'.center(256, '=') would get the point across without smashing out the right margin